### PR TITLE
[codestyle] Remove those useless parentheses

### PR DESCRIPTION
**Pull request by SrijanJha28**

### CODE REVIEW

File: xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java [hunk 1]
```diff
@@ -242,7 +242,7 @@ public boolean equals(Object object)
                 SyntaxType syntaxType = (SyntaxType) object;
                 // Note that the name isn't part of the hashCode computation since it's not part of the Syntax type's
                 // identity.
-                result = (getId() == syntaxType.getId() || (getId() != null && getId().equals(syntaxType.getId())));
+                result = (getId() == syntaxType.getId() || getId() != null && getId().equals(syntaxType.getId()));
```
Comment (vmassol) [inline xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java]: I think you're removing the wrong parenthesis ;)

File: xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java [hunk 2]
```diff
@@ -239,10 +239,10 @@ public boolean equals(Object object)
                 result = false;
             } else {
                 // Object must be Syntax at this point.
-                SyntaxType syntaxType = (SyntaxType) object;
+                SyntaxType syntaxType = SyntaxType object;
```
Comment (vmassol) [inline xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java]: This can't work for sure :)

Comment (vmassol) [reply xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java]: You always need to build the project to ensure it works and pass the tests...

File: xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java [hunk 3]
```diff
@@ -239,10 +239,10 @@ public boolean equals(Object object)
...
                 // Note that the name isn't part of the hashCode computation since it's not part of the Syntax type's
                 // identity.
-                result = (getId() == syntaxType.getId() || (getId() != null && getId().equals(syntaxType.getId())));
+                result = (getId() == syntaxType.getId() || (getId() != null && getId().equalssyntaxType.getId()));
```
Comment (vmassol) [inline xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java]: This also can't work... Please don't create PRs that don't pass the build ;)

Comment (SrijanJha28) [reply xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java]: sorry :)

**Comment by SrijanJha28** [comment]
Done :)

**Comment by vmassol** [comment]
Thanks

**Pull request merged**